proposals/interface-types/working-notes/scenarios/scenarios.md

```diff
@@ -0,0 +1,18 @@
+# A suite of adapter scenarios
```

Comment:
nit: might be good to capitalize "Scenarios.md" to match "WorkingNotes.md" and distinguish this file from all the other (lower-cased) files in the directory since it's a descriptor for the whole dir.

Reply:
Good suggestion.